libdnf/utils/utils.cpp

```diff
@@ -172,41 +172,48 @@ bool haveFilesSameContent(const char * filePath1, const char * filePath2)
     return ret;
 }
 
+static bool
+saveFile(const char * filePath, const char * newFileContent, unsigned long newFileContentLen)
+{
+    int fd = -1;
+    if ((fd = open(filePath,  O_WRONLY | O_CREAT | O_TRUNC, 0644)) == -1)
+        return false;
+    lseek(fd, 0, SEEK_SET);
```

Comment:
Is `lseek` needed? File was opened with `O_TRUNC` flag. So, I beleave that it is empty and position in file is zero..